tests/acceptance/standard/test_integrations.py

```diff
@@ -68,6 +68,39 @@ def test__if_delete_works(self, project):
             integration = project.integrations.get(integration_name)
             assert integration.active is False
 
+    def test__if_delete_without_create_works(self, other_project):
+        for integration_name in ["jira", "slack"]:
+            try:
+                other_project.integrations.get(integration_name)
+            except GitlabGetError as e:
+                assert (
+                    e.response_code == 404
+                ), f"Integration {integration_name} should not exist, but got a different error: {e}"
+            else:
+                assert False, f"Integration {integration_name} should not exist."
```

Comment:
I believe the `other_project` fixture's scope is class level. There's another fixture named `project_for_function`. If we use that fixture instead, we shouldn't need this extra setup here trying to validate the integrations don't exist before we start the tests - the project will be created fresh for this function without any pre-configured settings.